### ISSUE TITLE
Rails 5 support

### ISSUE THREAD

**Issue by rafaelfranca** [plataformatec/simple_form#1395]
This is an issue to track the other issues that are required to be have support to Rails 5.

**Comment by terry90** [comment]
`undefined method `number?' for #<ActiveRecord::ConnectionAdapters::PostgreSQLColumn:0x007fd63ddc2ad8>`

On a `f.input :field`

**Comment by fanta** [comment]
@terry90 glad I'm not the only one
```
ActionView::Template::Error (undefined method `number?' for #<ActiveRecord::ConnectionAdapters::MySQL::Column:0x007fd8bb68e110>

**Comment by fanta** [comment]
In the mean time, we can add `maxlength: 100` or whatever number to our inputs. What I found is that `active_record/connection_adapters/column.rb` does not have a `number?` method which is being used in `simple_form-3.2.1(or your version)/lib/simple_form/inputs/base.rb` in the `decimal_or_float?` method like this: `column.number? && column.type != :integer`, if you change that line perhaps to `column.scale && column.type != :integer` it works, but I'm not sure if using `column.scale` is the best way to know if it has a decimal point or not.

**Comment by rafaelfranca** [comment]
I should make clear that I'm not going to focus on fixing those issues for a while so if people want to investigate and open pull requests please go ahead.

**Comment by fuzzygroup** [comment]
>>  should make clear that I'm not going to focus on fixing those issues for a while so if people want to investigate and open pull requests please go ahead.

Can you define "a while"?  I don't know if you mean days, weeks or Performatec just isn't interested.  Thanks!

**Comment by rafaelfranca** [comment]
I can't speak for Plataformatec but I'm not using simple form for more than
3 years now and it is not my top priority. I reserved a night per week to
work on simple form but I may take weeks to actually do any work on the
rails 5 compatibility issue.

**Comment by fuzzygroup** [comment]
Ah.   Now worries then.  Thank you for the quick reply.

**Comment by afroniquely** [comment]
Anyone has other fixes on this issue? I got the same warning as above

**Comment by lucasmazza** [comment]
Hey everyone, please see https://github.com/plataformatec/simple_form/pull/1342#issuecomment-242468917 and try out Simple Form `3.3.0`. If we don't got any new issues regarding Rails 5 we can close this one in a couple of days.

**Comment by rafaelfranca** [comment]
:+1:

**Comment by thomasfedb** [comment]
Has this issue been resolved? If simple_form works with Rails 5 would be great to mention that here and close this issue.

**Comment by murb** [comment]
5.1 issues have already been solved, so I guess 5.0 support is hardly an issue at this moment.

**Comment by lucasmazza** [comment]
@thomasfedb @murb yes - latest release is both 5.0 and 5.1 compatible.
